packages/components/slider/src/use-slider-thumb.ts

```diff
@@ -101,16 +109,19 @@ export function useSliderThumb(props: UseSliderThumbProps) {
   };
 
   const getTooltipProps = () => {
-    const value = state.values[index ?? 0];
+    const value = numberFormatter
+      ? numberFormatter.format(state.values[index ?? 0])
+      : state.values[index ?? 0];
 
     const content =
       renderOutput && typeof renderOutput === "function" ? renderOutput(`${value}`) : value;
 
     return {
       ...tooltipProps,
-      content,
-      isOpen: isHovered || isDragging,
-      placement: isVertical ? "right" : "top",
+      placement: tooltipProps?.placement ?? isVertical ? "right" : "top",
+      content: tooltipProps?.content ?? content,
```

Comment:
Just move the `content: content,` befor the `...tooltipProps`

Reply:
I added this way to be more explicit, It took me some time to fix a bug because of the `...tooltipProps`